### PR TITLE
HM-4 Created DTOs

### PR DESCRIPTION
**Pull request by Sisi-06**

### CODE REVIEW

File: HotelManagerMVC/HotelManager.Shared/Dtos/ReservationDto.cs
Comment (Petkov06) [inline HotelManagerMVC/HotelManager.Shared/Dtos/ReservationDto.cs:1]: нещо липсва
не е ок!

**Review by Petkov06 — COMMENTED** [review]
погледни ENTITY-to Reservation

**Review by ivomihailov06 — COMMENTED** [review]
Added property is good and the new function is correct.

**Review by ivomihailov06 — COMMENTED** [review]
ОК

**Review by ectky — APPROVED** [review]

**Pull request merged**